python/pysmurf/client/util/smurf_util.py

```diff
@@ -2415,10 +2415,42 @@ def get_hemt_drain_current(self):
 
         # on cryostat card
         hemt_Vd_series_resistor=200  #Ohm
```

Comment:
Not your fault but this is a hardcode we should fix.  Not gonna hold up this PR but created an issue to address here https://github.com/slaclab/pysmurf/issues/311.

Reply:
Resolving because I opened up a separate issue for this.